.github/workflows/pulsar-ci-flaky.yaml

```diff
@@ -135,19 +151,22 @@ jobs:
         uses: ./.github/actions/copy-test-reports
 
       - name: Create Jacoco reports
+        if: ${{ needs.preconditions.outputs.collect_coverage == 'true' }}
         run: |
           $GITHUB_WORKSPACE/build/pulsar_ci_tool.sh create_test_coverage_report 
           cd $GITHUB_WORKSPACE/target
-          zip -qr jacoco_test_coverage_report_flaky.zip jacoco_test_coverage_report
+          zip -qr jacoco_test_coverage_report_flaky.zip jacoco_test_coverage_report || true
```

Comment:
Why do we ignore zip command failure here? ditto others.

Reply:
I came across a build where it had failed in pulsar-ci-flaky.yaml , https://github.com/apache/pulsar/actions/runs/3994116198/jobs/6851473755#step:12:17 . The reason is unknown. I didn't want the whole build to fail if that happens.